site/src/content/docs/commands/zarf_package_remove.md

```diff
@@ -24,6 +24,7 @@ zarf package remove { PACKAGE_SOURCE | PACKAGE_NAME } --confirm [flags]
       --components string           Comma-separated list of components to remove.  This list will be respected regardless of a component's 'required' or 'default' status.  Globbing component names with '*' and deselecting components with a leading '-' are also supported.
       --confirm                     REQUIRED. Confirm the removal action to prevent accidental deletions
   -h, --help                        help for remove
+  -n, --namespace string            [Alpha] Override the namespace for package removal. Applicable only to packages deployed using the namespace override flag.
```

Comment:
If we're adding the `-n` shorthand we should add it to the other commands. Also we should just say the "namespace flag" instead of "namespace override flag" imo.